code/datums/components/jetpack.dm

```diff
@@ -72,81 +84,79 @@
 /datum/component/jetpack/proc/setup_trail(mob/user)
 	if(trail)
 		QDEL_NULL(trail)
-
 	trail = new effect_type
 	trail.auto_process = FALSE
 	trail.set_up(user)
 	trail.start()
 
-/datum/component/jetpack/proc/activate(datum/source, mob/user)
+/datum/component/jetpack/proc/activate(datum/source, mob/new_user)
 	SIGNAL_HANDLER
 
 	if(!check_on_move.Invoke(TRUE))
 		return return_flag
 
+	user = new_user
```

Comment:
what happens if the user is deleted before we deactivate?

Reply:
it is only stored for process(), unless a tick happens between deletion and deactivation it should be fine? I'll add nulling used in Destroy() but unless I am misunderstanding how GC/harddels work, beyond that change there should be no issues